pkgs/development/python-modules/aioquic/default.nix

```diff
@@ -17,6 +18,19 @@ buildPythonPackage rec {
     sha256 = "sha256-7ENqqs6Ze4RrAeUgDtv34+VrkYJqFE77l0j9jd0zK74=";
   };
 
+  patches = [
+    # This patch is here because it's required by the next patch.
+    (fetchpatch {
+      url = "https://github.com/aiortc/aioquic/commit/3930580b50831a034d21ee4689362188b21a4d6a.patch";
+      hash = "sha256-XjhyajDawN/G1nPtkMbNe66iJCo76UpdA7PqwtxO5ag=";
+    })
+    # Fixes test failure due pyopenssl==22
+    (assert lib.versions.major pyopenssl.version == "22"; fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/aiortc/aioquic/pull/349.patch";
+      hash = "sha256-AjW+U9DpNXgA5yqKkWnx0OYpY2sZR9KIdQ3pSzxU+uY=";
```

Comment:
The PR isn't merged yet so the patch can still change. Use the commit-specific patch instead

Reply:
Applied suggestion.